libraries/GCS_MAVLink/MissionItemProtocol.cpp

```diff
@@ -36,6 +36,13 @@ void MissionItemProtocol::handle_mission_count(
     const mavlink_mission_count_t &packet,
     const mavlink_message_t &msg)
 {
+    // need mavlink2 to do mission types other than mission:
```

Comment:
Maybe extract this as a helper/macro? Seems to be here verbatim 3 times.

Reply:
Made a helper for it.